README.md

```diff
@@ -1,34 +1,43 @@
 
-## Usage
+## Using the [AdyenDev](https://www.postman.com/adyendev) workspace
 
-The preferred way to use these files is to fork the [Adyen APIs](https://adyendev.postman.co/workspace/Adyen-APIs~a8d63f9f-cfc7-4810-90c5-9e0c60030d3e) Postman workspace. This way you will be notified of new versions and changes.
-However, in case you want to experiment, or simply want to keep your experiments private, you can also take any of the json files in this repository and upload them yourself.
+The preferred way to use these files is to fork the [Adyen APIs](https://adyendev.postman.co/workspace/Adyen-APIs~a8d63f9f-cfc7-4810-90c5-9e0c60030d3e) Postman workspace. 
 
-![Importing a json file as a Postman collection](./images/import-postman.gif)
+![Creating a fork for a Postman collection](./images/create-fork.gif)
+
+
+This way you will be notified of new versions and changes per email and in the UI directly.
+
+![Receiving the email of a collection changed](./images/postman-email.png)
 
+## Importing the JSON files
 
-## Contributing
+However, in case you want to experiment, want to keep your experiments private or want to use an older version of any of our APIS, you can also take any of the json files in this repository and upload them yourself.
 
-We encourage you to contribute to our repository. Find out more in our [contribution guidelines](https://github.com/Adyen/.github/blob/master/CONTRIBUTING.md).
-However, for anything related to our APIs, the correct repository to contribute to would be [adyen-openapi](https://github.com/adyen/adyen-openapi).
+![Importing a json file as a Postman collection](./images/import-postman.gif)
 
 ## Setup
 
-* `generateAll.sh` generates all postman API collection files for all versions of the OpenAPI definitions and places them in the postman folder. It is meant to run inside the [dedicated docker image](https://github.com/gcatanese/openapi-generator-postman-v2/).
-* `runDocker.sh` is the script that actually runs `generateAll.sh` inside the docker image. 
+Before running API calls, you will have to set some variables. Those depend on which API you are using, but you will recognize them because they are between braces (`{{X-API-Key}}`).
+Notably, you will find those in the `Authorization` as well as the `Variables` tabs in each collection. 
+
+![Variables to set in my collection](./images/collection-variables.png).
+
+Instead of directly modifying them there, we recommend you instead to create [Environments](https://learning.postman.com/docs/sending-requests/managing-environments/). This way, you can switch from test to production without much hassle when running your API calls. Environments are also nice to protect secret values, like your API Keys. 
+
+To do this, create a new environment using the corresponding left tab in the Postman UI, and override the variables there.
```

Comment:
We have also a `Adyen APIs` environment in AdyenDev which devs can fork and make it private

Reply:
oh nice, I didn't know!